### DISCUSSION TITLE
如何保存任务记录，不是日志记录。

### DISCUSSION THREAD

**Discussion by loomee** [AlistGo/alist#4986] [Q&A]
Alist版本3.25,系统是win11.
我挂在了好几个网盘，同时也把电脑上的第二个硬盘挂在到了alist。
我在alist的webUI里把百度网盘中的文件夹复制到alist挂在的本地硬盘里，
然而因为文件数量多，网盘又限速，所以需要好几天才能完成所有。
在这过程中，我可能遇到意外重启电脑，
当我重新进入Alist的webUI里，发现任务里全是空的，而实际上任务并没有全完成。
同时还有些出错失败的任务需要重试，这些记录也都没有了。
如果没有意外重启，那些记录全都存在。
我尝试过在Alist的配置文件里，启用日志记录，
然而重启Alist后，webUI里任务记录依然全空。
包括之前的alist版本v3.24和v3.23也都如此。
我想知道需要修改哪个配置条目才能保存复制任务。
又或者这是个 new issue!

**Comment by loomee** [comment]
关于重启Alist或者重启电脑之后，Alist的任务记录全没了。
主要是复制任务出错需要重试的还有进行中的那些都没有了。 http://127.0.0.1:5244/@manage/tasks/copy 
查阅过帮助文档没结果 https://alist.nn.ci/zh/config/configuration.html 
搜索过 issues 也搜索过 discussion 都没找到答案。
不知道这是程序本身不能保存记录，还是在某个地方有开关呢

**Comment by xhofe** [comment]
暂不支持
